components/date_picker/index.js

```diff
@@ -1,5 +1,3 @@
-export default from './DatePicker';
```

Comment:
If you rebase with `dev` this should not be a problem. In any case we have to pull it out of the PR